rhel8/profiles/ism_o.profile

```diff
@@ -0,0 +1,146 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Information Security Manual (ISM) Official'
+
+description: |-
+  This profile contains configuration checks for Red Hat Enterprise Linux 8
+  that align to the Australian Cyber Security Centre (ACSC) Information Security Manual (ISM)
+  with the Attorney-General’s Department (AGD)’s applicability marking of OFFICIAL.
+
+  A overview and list of Cyber security guidelines of the 
+  Information Security Manual can be found at the ACSC website:
+
+  https://www.cyber.gov.au/ism
+
+extends: e8
+
+selections:
+
+  ## Operating system configuration
+  ## Identifiers 1491
+  - no_shelllogin_for_systemaccounts
+
+  ## Local administrator accounts
+  ## Identifiers 1410
+  - accounts_password_all_shadowed
+
+  ## Content filtering & Anti virus
+  ## Identifiers 1341 / 1034 / 1417 / 1288
+  - package_aide_installed
+
+  ## Software firewall
+  ## Identifiers 1416
+  - configure_firewalld_ports
+  - configure_firewalld_rate_limiting
+  - firewalld_sshd_port_enabled
+  - set_firewalld_default_zone
+
+  ## Endpoint device control software
+  ## Identifiers 1418
+  - package_usbguard_installed
+  - service_usbguard_enabled
+  - usbguard_rules_not_empty_not_missing
+
+  ## Authentication hardening
+  ## Identifiers 1546 / 0974 / 1173 / 1504 / 1505 / 1401 / 1559 / 1560
+  ## 1561 / 0421 / 1557 / 0422 / 1558 / 1403 / 0431
+  - disable_host_auth
+  - require_emergency_target_auth
+  - require_singleuser_auth
+  - sebool_authlogin_nsswitch_use_ldap
+  - sebool_authlogin_radius
+  - sshd_disable_kerb_auth
+  - sshd_set_max_auth_tries
+  - sssd_enable_smartcards
+  - accounts_password_minlen_login_defs
+  - accounts_password_pam_minclass
+  - accounts_password_pam_minlen
+  - accounts_password_pam_pwquality
```

Comment:
This is similar, Th rule `accounts_password_pam_pwquality` doesn't exist, it just a check that's embedded in other `accounts_password_pam_.*` rules.

Reply:
Updated @jan-cerny https://github.com/wcushen/content/commit/f90d535a32aa2e569b70c9998bbda8e58facca55